submissions/AnnaStoyano/pre-oop/index.js

```diff
@@ -0,0 +1,90 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Lucy',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'guf',
+   friends: ['Luck']
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Murzick',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'meow',
+   friends: ['Elza','Nicky']
+};
+
+const woman = {
+   species: 'human',
+   name: 'Anna',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Hello!',
+   friends: ['Sergey','Kate','Karina']
+}
+
+const man = {
+   species: 'human',
+   name: 'Vlad',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi!',
+   friends: ['Sergey','Anton','Vlad']
+}
+
+const catWoman = {
+   species: 'human',
+   name: 'Liza',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
+   friends: ['Batman']
+};
+
+
+const inhabitants = [man,woman,cat,dog,catWoman];
+
+const properties = ['name','species','gender','legs','hands','saying','friends'];
+let output = inhabitants.map(inhabitant=>{
+   let match = properties.map(property=>{
+      return `<b>${property}:</b> ${inhabitant[property]}`
+   });
+   return match.join('\t')
+})
```

Comment:
Chaining:
```
let a = arr.map(...);
return a.join(...);

// versus
return arr.map(...).join(...);
```

Arrow functions:
```
arr.map(e => {
  let a = e*2;
  return a;
})

// versus
arr.map(e => e*2);
```
Please let me know if the above gives you hints on how to make the code even more concise.